gatling/user-files/utils/create-users.js

```diff
@@ -0,0 +1,47 @@
+const
+  rp = require('request-promise'),
+  host = process.argv[2] || 'localhost';
+
+const doit = (id) => {
+  for (let it = 1; it < 100; ++it, ++id) {  
+    try {
+      let options = {
+        method: 'POST',
+        uri: `http://${host}:7512/users/${id}/_create?refresh=wait_for`,
+        body: {
+          'content': {
+            'profileIds': ['default'],
+            'fullname': 'John Doe'
+          },
+          'credentials': {
+            'local': {
+              username: id,
+              password: 'test'
+            }
+          }
+        },
+        forever: true,
+        json: true
+      };
+      rp(options);
+    }
+    catch (error) {
+      // eslint-disable-next-line no-console
+      console.error(error.message);
+    }
+  }
+};
+const create_users = async () => {
+  let id = 1;
+  for (let it = 1; it <= 20; ++it) {
+    try {
+      await doit(id);
+      id += 100;
```

Comment:
This is a [magic number](https://en.wikipedia.org/wiki/Magic_number_%28programming%29). If it was only used here it wouldn't be so bad, but for that code to work, the same number must be used by the `doit` function above.

This is a bad practice. Instead, you should create a new constant (for instance: `const usersBundleSize = 100;`), used by both functions. That way, modifying the value will affect all parts of the code using it, preventing otherwise dangerous (and hard to debug) side effects.

Also, the `id` variable here can be removed if you invoke the `doit` method like this:

```js
await doit(it * usersBundleSize);
```